sdk/core/azure-core-experimental/src/main/java/com/azure/core/experimental/http/policy/ArmChallengeAuthenticationPolicy.java

```diff
@@ -69,9 +69,7 @@ public ArmChallengeAuthenticationPolicy(TokenCredential credential,
     public Mono<Boolean> authorizeRequestOnChallenge(HttpPipelineCallContext context, HttpResponse response) {
         return Mono.defer(() -> {
             String authHeader = response.getHeaderValue(WWW_AUTHENTICATE);
-            if (!(response.getStatusCode() == 401 && authHeader != null)) {
```

Comment:
Can `ArmChallengeAuthenticationPolicy` be removed from `experimental` package? It is already copied over to`azure-core-management`.
cc: @weidongxu-microsoft, @g2vinay

Reply:
Yes, it can be removed.

Reply:
yeah, we can remove it.